src/webview/direct-connect-form.html

```diff
@@ -39,44 +39,75 @@ <h1>Create a new connection (⚠️ experimental)</h1>
                 required
               />
             </div>
-            <span class="description">A comma-separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster</span>
+            <span class="description"
+              >A comma-separated list of host/port pairs to use for establishing the initial
+              connection to the Kafka cluster</span
+            >
           </div>
         </fieldset>
         <fieldset>
           <legend>Schema Registry (optional)</legend>
           <div class="input-container">
             <div>
               <label for="uri" class="label">Schema Registry URL</label>
-              <input
-                class="input"
-                id="uri"
-                name="uri"
-                type="url"
-              />
+              <input class="input" id="uri" name="uri" type="url" />
             </div>
             <span class="description">The URL of the Schema Registry to use for serialization</span>
+          </div>
```

Comment:
This is the actual culprit of the error 
`SyntaxError: Unexpected closing tag "fieldset".` 

The rest of the changes are just formatting line length